src/tests/createHeadlessForm.test.js

```diff
@@ -9,6 +9,7 @@ import {
   schemaInputTypeText,
   schemaInputTypeRadioDeprecated,
   schemaInputTypeRadioString,
+  schemaInputTypeRadioStringY,
```

Comment:
Hey Dilvane! I won't be able to review this as I'm catching up from PTO. @brennj instead can you cover me here?